scripts/entrypoint.sh

```diff
@@ -47,6 +47,11 @@ Arguments:
     dependencies) in parse-mode. This can catch problems with packages
     that do not include tests, or whose testing is too narrow.
 
+  --set-safe-directory auto
+
+    Add repository path as safe.directory to the Git global config
+    used in the container. auto means done when pkg is ".".
```

Comment:
Arne did you consider not adding an option and just running `git config` whenever `pkgurl` is "."? I suspect you want to give people a way to opt out in case it ever becomes a problem? I'm fine with that, just wanted to check. If we keep the option, I'd make this `--set-safe-directory <yes|no>`, defaulting it to `yes`, since it effectively acts as a boolean.

Reply:
I'm happy making it implicit and removing the option. There's probably no user who'd ever opt-out. If they come, we can add something then.

The fact that it's `auto` indicates that I wasn't quite settled on where this should go.